.github/codeql.yml

```diff
@@ -0,0 +1,55 @@
+name: "Code scanning - action"
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 19 * * 0'
```

Comment:
maybe I'm reading this wrong, but this seems to say that codeql will run daily at 7pm whether there are any code changes or not, and will not run on PRs or pushes? Is that right? If so, is that what we want?

Reply:
This is the default setting. I believe it does a scheduled scan as well as a quick diff vs the commit being built. I can look into the specifics more, but this is the stock setting, and is safe to leave as is for now.